docs/model/person.md

```diff
@@ -1,3 +1,49 @@
+A **Person** entry provides detailed information about individuals involved in research projects, including their affiliations, roles, and contributions. Accurate person records are essential for tracking collaborations, grant participation, working group involvement, and expertise within scientific initiatives.
+
+This section outlines how to create a valid person entry, ensuring that all necessary details are captured to support identification, categorization, and data integration within  MC<sup>2</sup> Center-supported projects.
+
+### **Why You Should Contribute Person Entries**
+Contributing person entries enhances collaboration by providing clear visibility into expertise, roles, and project involvement within research teams. It helps connect researchers and contributors across projects, facilitates networking opportunities, and allows stakeholders to easily identify key personnel for collaboration or consultation. Maintaining up-to-date profiles also supports transparency and improves recognition of contributions to research initiatives.
+
+#### **Who Should Be Contributing Person Entries?**
+1. **Researchers and Scientists** – Document your roles, publications, and collaborations to increase visibility within your field and enable others to reach out for partnerships.  
+2. **Project Leads and Investigators** – Ensure key contributors to your projects are credited and easily searchable for collaboration and reporting purposes.  
+3. **Research Administrators** – Keep records of team members, roles, and working group participation to manage projects efficiently and facilitate compliance reporting.  
+4. **Consortium Members** – Share your expertise and involvement across multiple projects or institutions to encourage cross-disciplinary collaboration and integration of research efforts.  
+5. **Data Managers** – Link researchers to datasets, tools, and resources they have contributed to ensure traceability and acknowledgment of contributions.  
+
+## Download Template
```

Comment:
Downloading this template is not currently our workflow for curating Person entries. Until we have a revised process in place, the individuals listed under "Who should be contributing person entries?" should solely be directed to the form here: https://docs.google.com/forms/d/e/1FAIpQLScuFHETXIdfe0HPqkFQ7jx3wIIwBRqaqIXH_ZM5f2DWvNQa2g/viewform

Based on information provide in the form, including specific consent collected, the MC2 Center would be the ones filling out the PersonView template